server/service/transport.go

```diff
@@ -227,6 +227,11 @@ func hostListOptionsFromRequest(r *http.Request) (fleet.HostListOptions, error)
 		hopt.SoftwareIDFilter = &sid
 	}
 
+	disableFailingPolicies := r.URL.Query().Get("disable_failing_policies")
+	if disableFailingPolicies != "" {
+		hopt.DisableFailingPolicies = disableFailingPolicies == "1" || disableFailingPolicies == "true"
```

Comment:
Nit but we may want to consider `strconv.ParseBool` everywhere we need this, for consistency: https://pkg.go.dev/strconv#ParseBool

Reply:
Ah! TIL, yes, changing to that.